### ISSUE TITLE
Side effect of handleClickCol is not ideal for pagination usage

### ISSUE THREAD

**Issue by tristaaan** [dasDaniel/svelte-table#42]
I'm using the this table for paginating lots of data. When the headers are clicked on I am sending an API call from which results will already be sorted and so the table need not do any sorting itself. I think it's an interesting side effect that `handleClickCol` also sorts as well as triggering the `clickCol` event. I'm interested in preventing a call to `updateSortOrder(...` if a certain flag is passed in `preventSortOnClickCol` maybe?

I'm willing to implement this and update relevant documentation but I thought I would ask for some feedback first.
The method in question:
```
  const handleClickCol = (event, col) => {
    updateSortOrder(col.key)  # planning on wrapping these 
    sortBy = col.key;         # two lines in a conditional
    dispatch('clickCol', {event, col, key:col.key} ); 
  }
```

**Comment by dasDaniel** [comment]
that could work, but I think it is a bit counter-intuitive

The click functionality is bound within `{#if col.sortable}` so that means that if you'd like the functionality you would enable sorting  only to disable it.  

https://github.com/dasDaniel/svelte-table/blob/develop/src/SvelteTable.svelte#L155-L164


I'm thinking it might make more sense to add the `on:click={(e) => handleClickCol(e, col, false)}` handler to the `else` block where the `false` would prevent the sort update.

I haven't tested this so I'm not sure it would work, but it if does, I think that would keep the API the same. I'm planing on doing a bit of an overhaul soon so I'd like to keep the API the same while I'm trying to figure out what to do with this.

**Comment by tristaaan** [comment]
fixed with #43
